alf/layers.py

```diff
@@ -262,6 +264,7 @@ def __init__(self,
         # get the argument list with vals
         self._kwargs = copy.deepcopy(locals())
         self._kwargs.pop('self')
+        self._kwargs.pop('use_bn')
```

Comment:
I suppose you did this because ParallelFC does not support use_bn. But instead of silently ingoring the problem, we should report error.